logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -28,9 +28,9 @@
     public static final String VERSION = "@version";
     public static final String VERSION_ONE = "1";
 
-    private static final ObjectMapper mapper = new ObjectMapper();
-
-    // TODO: add metadata support
+    private static final Logger DEFAULT_LOGGER = new StdioLogger();
+    private transient final ObjectMapper mapper = new ObjectMapper();
+    private transient static Logger logger = DEFAULT_LOGGER;
```

Comment:
I am not sure. at this point this is only mutated in the tests/specs. there is not really *currently* any point in changing the logger at runtime. for now this is what make it work with the less overhead, this is certainly something we can improve.